src/pages/PokemonGuess.tsx

```diff
@@ -16,56 +17,120 @@ export function PokemonGuess() {
     function setupGame() {
         const choices = shuffle(pokemons).slice(0, 4);
         const [correct] = choices;
-        console.log(shuffle(choices), correct);
         setCorrectPokemon(correct);
         setPokemonChoices(shuffle(choices));
     }
 
+    function handleChoice(pokemonName: string) {
+        const updated = pokemonChoices.map((item) => {
+            if (item.pokemonName == pokemonName && pokemonName == correctPokemon?.pokemonName) {
+                return {
+                    ...item,
+                    colour: "green"
+                };
+            } else if (item.pokemonName == pokemonName) {
+                return {
+                    ...item,
+                    colour: "red"
+                };
+            } else {
+                return item;
+            }
+        });
+        setPokemonChoices(updated);
+        if (pokemonName == correctPokemon?.pokemonName) {
+            setIsReveal(true);
+            //set up a new game
+            setTimeout(async () => {
+                await fetchPokemons();
+            }, 3000);
+        }
+    }
+
     useEffect(() => {
         if (pending) {
             return;
         }
         setupGame();
+        setIsReveal(false);
     }, [pokemons]);
 
     return (
-        <div>
-            <h1>Who is that Pokemon!!!!</h1>
-
+        <Container>
+            <Title>Who is that Pokemon ?</Title>
             <ImageContainer>
-                <HiddenImage>
+                <HiddenImage $isOpen={isReveal} style={{ filter: `brightness(${isReveal ? 1 : 0})` }}>
                     <PokemonImage href={correctPokemon?.pokemonSprite} />
                 </HiddenImage>
             </ImageContainer>
-            <div>
+            <Choices>
                 {pokemonChoices &&
                     pokemonChoices.map((item: PokemonGuessInfo, idx) => (
-                        <div key={idx}>
-                            <h1>PokemonName: {item.pokemonName}</h1>
-                        </div>
+                        <ChoiceButton
+                            key={idx}
+                            onClick={() => !isReveal && handleChoice(item.pokemonName)}
+                            style={{ background: item.colour }}
+                        >
+                            <h1> {item.pokemonName}</h1>
+                        </ChoiceButton>
                     ))}
-            </div>
-        </div>
+            </Choices>
+        </Container>
     );
 }
 
+const Container = styled(ContainerPrototype)`
```

Comment:
There is a small issue with total width and height of the css where page size is slightly bigger than screen size